src/core/create_compilers/RollupCompiler.ts

```diff
@@ -39,10 +39,14 @@ export default class RollupCompiler {
 				this.chunks.push(chunk);
 			},
 			transform: (code: string, id: string) => {
-				if (/\.css$/.test(id)) {
-					this.css_files.push({ id, code });
-					return ``;
+				if (!/\.css$/.test(id)) return;
+				
+				if (/styleInject/.test(code)) {
```

Comment:
There's a lot to comprehend regarding this bug and I'm not sure I totally understand it yet because I'm not that familiar with PostCSS, etc. Looking for `styleInject` seems too specific to this usecase though and I think it'd be better handled in a more generic way. The first line of this function is checking to make sure that the code is CSS. Perhaps the issue here is that you actually have JavaScript in a file with a `.css` extension? It seems weird to me that you would have that, but perhaps the fix should be to check if the file contents are JavaScript instead of CSS? But I'm not quite sure a good way of doing that. Looking for something like `export` seems a bit better, but also seems like it could cause problems. E.g. what if the user has some data export functionality on their site and has a CSS class named `.export`?